html/credentials.inc.example.php

```diff
@@ -1,9 +0,0 @@
-<?php
-
-# DB credentials
-define("DBHOST", "localhost");
-define("DBNAME", "test");
-define("DBUSER", "test");
-define("DBPASS", "test");
-
-?>
```

Comment:
You shouldn't delete this. This is for providing future developers with a template for a credential file.